src/components/MAPI/clusters/guides/CreateClusterGuide.tsx

```diff
@@ -0,0 +1,90 @@
+import { Text } from 'grommet';
+import * as docs from 'lib/docs';
+import LoginGuideStep from 'MAPI/guides/LoginGuideStep';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Providers } from 'shared/constants';
+import { PropertiesOf } from 'shared/types';
+import CLIGuide from 'UI/Display/MAPI/CLIGuide';
+import CLIGuideAdditionalInfo from 'UI/Display/MAPI/CLIGuide/CLIGuideAdditionalInfo';
+import CLIGuideStep from 'UI/Display/MAPI/CLIGuide/CLIGuideStep';
+import CLIGuideStepList from 'UI/Display/MAPI/CLIGuide/CLIGuideStepList';
+
+interface ICreateClusterGuideProps
+  extends Omit<React.ComponentPropsWithoutRef<typeof CLIGuide>, 'title'> {
+  provider: PropertiesOf<typeof Providers>;
+  clusterName: string;
+  organizationName: string;
+}
+
+const CreateClusterGuide: React.FC<ICreateClusterGuideProps> = ({
+  provider,
+  clusterName,
+  organizationName,
+  ...props
+}) => {
+  return (
+    <CLIGuide
+      title='Create a cluster via the Management API'
+      footer={
+        <CLIGuideAdditionalInfo
+          links={[
+            {
+              label: 'kubectl gs plugin installation',
+              href: docs.kubectlGSInstallationURL,
+              external: true,
+            },
+            {
+              label: 'kubectl gs template cluster command',
+              href: docs.kubectlGSTemplateClusterURL,
+              external: true,
+            },
+            {
+              label: 'Cluster CRD schema',
```

Comment:
Also added a link to the `Cluster` CRD, to keep it consistent with the organizations guides